test/core/end2end/generate_tests.bzl

```diff
@@ -408,18 +412,14 @@ def grpc_end2end_tests():
             "//test/core/compression:args_utils",
         ],
     )
-
     for f, fopt in END2END_FIXTURES.items():
+        # TODO(hork): try removing this to see if we no longer need the raw bin.
```

Comment:
I don't think we need this anymore.

Reply:
Exactly, we may not, that's why I left the TODO. I want to remove it separately and see who or what complains, but not block this PR. I have personally used these binaries for running tests under gdb, and I'm not sure if anyone else does as part of their workflow, or has automation around it. Cmake tests may use it, I haven't verified (and didn't want to make that a requirement for this PR, since it's a tangential concern).